scripts/ci/install-minikube.sh

```diff
@@ -45,7 +47,20 @@ check_minikube() {
         sudo systemctl start docker
 }
 
+patch_iptables() {
+        # HACK HACK HACK: kube-proxy passes iptables-restore flag -w5 which is
+	# ignored in v1.6.1 (FC27) but fails v1.6.2 (FC28). Issue can be found
+	# https://github.com/NixOS/nixpkgs/issues/35544, it's patch has not yet
+	# been released in default kubernetes server used by minikube.
+        . /etc/os-release
+        if [ "$ID" == "fedora" -a "$VERSION_ID" == "28" ]; then
+		sudo rm -rf /usr/local/sbin/iptable-restore
```

Comment:
could you try your patch on a fresh fc28 ..

What do you expect of ?
sudo iptables-restore -w5

hint: you should check /sbin /usr/sbin